Attributes.cs

```diff
@@ -0,0 +1,32 @@
+﻿using DSharpPlus;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ThomasBot
+{
+    class Attributes
+    {
+        /// <summary>
+        /// Defines that usage of this command is restricted to Guilds.
+        /// </summary>
+        [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
+        public sealed class GuildOnly : CheckBaseAttribute
+        {
+            public override Task<bool> ExecuteCheckAsync(CommandContext ctx, bool help)
+                => Task.FromResult(ctx.Guild != null || help);
+        }
+
+        /// <summary>
+        /// Defines that usage of this command is restricted to the bot-channel. Developers can override this.
+        /// </summary>
+        [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
```

Comment:
we do not restrict channel usage

Reply:
Channel usage is only restricted when a command is tagged with this attribute.
Currently, none are tagged with it.